### PR TITLE
Issue #88

### PR DESCRIPTION
**Pull request by nuageklow**
Created a README.md based on suggestions.

### CODE REVIEW

File: community_health_file.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+The new version of the website will include features that will help users:
+* share and recommend resources/tutorial links
+* find open source projects to work on
+* form study groups
+* collect learning paths and share them
+* pair programming -- find 1-hour pair programming partners for any topics in a much more easier way
+
+## **Role of Backend**
+The backend of CBV3 is an app which is composed of Django REST Framework that provides API endpoints which the frontend app (React) consumes.
```
Comment (lpatmo) [inline community_health_file.md]: Can you tweak this to be:

The backend is a Django app that provides API endpoints (built using Django REST Framework) that the frontend (a React app) consumes.

File: community_health_file.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+__Core dependencies of CBV3__
+| Dependencies | Version |
+| ------------ | :-----: |
+| [djangorestframework](https://github.com/encode/django-rest-framework) | 3.10.2 |
+| [coreapi](https://github.com/core-api/python-client) | 2.3.3 |
+| [drf-jwt](https://githbu.com/Styria-Digital/django-rest-framework-jwt) | 1.13.4 |
+
+## **Can I run the website application on my computer?**
+Of course you can. Follow the [instruction](contributing.md) to set up the Django API backend.
```
Comment (lpatmo) [inline community_health_file.md]: Can you change [instruction] to [instructions]? Thanks!

File: community_health_file.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+Set up using Docker is highly recommended.
+
+## **Have Questions about CBV3?**
+Check out [support.md](support.md) if you're stuck or have questions.
+
+## **Ways to Get Involved**
+Anyone is welcome to contribute and make the website better! You can:
+* Join our slack community in [here](https://codebuddies.org/slack)
```
Comment (lpatmo) [inline community_health_file.md]: Can you change to:

* Join our Slack community [here]

File: community_health_file.md (hunk 1 above)
Comment (nuageklow) [reply community_health_file.md]: just a quick question/clarification on some technical terms 
I've always thought Django is a framework that coders can write apps within the framework. Do the definitions of app and framework are similar in this case? 
I'm not too familiar with applications and frameworks let me know thank you :)

**Comment by nuageklow** [comment]
thank you @lpatmo for reviewing that changes are made based on suggestions above :)

File: community_health_file.md (hunk 1 above)
Comment (lpatmo) [reply community_health_file.md]: Good question! Django by itself is a framework, you're right! We're using a package called Django REST Framework to make our API. I called it a "Django app" in the sense that it is a web app, just as any sort of website is an app to me. I wasn't thinking of the technical definition of `app` inside Django per se. :) 

Maybe to be clearer:

The backend is a Django application that provides API endpoints (built using [Django REST Framework](https://github.com/encode/django-rest-framework)) that the frontend (a React app) consumes.

**Comment by BethanyG** [comment]
Mashing big, green button.  🎉

**Pull request merged**